README.md

```diff
@@ -150,3 +150,18 @@ Simply open the browser console and run the commands
 
 :warning: **NOTE**
 It is suggested to clear local storage and cookies after enabling and disabling authentication. It will help starting a fresh session.
+
+
+## New login approach
+Based on new requirements (see story _IPROD-189_) we need to use Keycloak to authenticate users.
+For this purposes *OAuth 2.0 authorization code flow* is used. 
+
+### Env variables for OAuth OIDC flow:
+- `AUTH_ENABLED` optional and disabled by default. To enable the previous auth flow set it to `true`
+- `UI_OIDC_LOGIN_REDIRECT_URL` - URL to redirect user to provide creds and grand permissions (Keycloack)
+- `OIDC_TOKEN_PROVIDER_URL` - URL to exchange _authorization code_ for _access token_ (providing also client ID and secret)
+- `MCM_CLIENT_ID` - MCM portal client id (in Keycloak)
```

Comment:
I suggest that we should rename them env-vars or configs that are `MCM_*` prefixed since we are trying to be OIDC provider neutral here (i.e. not tied into MCM or Keycloak), and suggest that we should generalise them.

Let me know your thoughts on this.